app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def accept_transition_info
+    user = User.new(id: current_user["user_id"])
```

Comment:
Would it make sense to try to hide this little bit of ugly use of `new` by creating a method to mask it? 

```
User.member(current_user['user_id']).accept_transition_user

# and in User:

def self.member(id)
  # BAD JSON API client ... BAD BAD BAD JSON API client
  new(id)
end
```

?

Reply:
Sure - fixed in 272bcd9